upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -47,6 +47,7 @@ function print_footer() {
 function print_section() {
     echo ""
     echo "======================================================================================"
+    echo "=========================$1=================================================="
```

Comment:
minor point - maybe a space around $1 for better readability, e.g. ...=== $1 ===...
and maybe remove a few === from the right side of $1 so that "Machine configuration pools" doesn't push the '===' to far to the right.
again, very minor points

Reply:
thanks, took care of it and it indeed makes output more readable